test/test_macho.rb

```diff
@@ -313,8 +321,15 @@ def test_change_install_name
 
       file.write(actual)
 
-      # compare actual and expected file hashes, to ensure file correctness
-      assert equal_sha1_hashes(actual, expected)
```

Comment:
I added it above, right above the creation of `act` and `exp`. I just realized that it's missing from `test_change_rpath`, though...